### PR TITLE
Added toast for successfull submission #33

### LINKED ISSUE #33: Add success message on successful form submission
Currently the successful message is being logged on console. It would be great to have to show up below the submission form.

Changes to be made on Submission.jsx component inside components folder.

image
A green message under the submit button is enough!

### PR DESCRIPTION
**Pull request by Yuvraj-Achrekar**
Added toast after submission in submission.jsx
Fixed #33

### CODE REVIEW

**Comment by plon-Susk7** [comment]
Hey @Yuvraj-Achrekar , this looks great. Could you also add a toast in case of failure?

File: src/components/Submission.jsx [hunk 1]
```diff
@@ -1,80 +1,94 @@
...
+				}
+			);
 
-            const data = await octokit.request('POST /repos/plon-Susk7/Job-Board/issues',{
-                owner: 'plon-Susk7',
-                repo: 'Job-Board',
-                title: 'New Project Submission',
-                body: `Project: ${github_url}\nCategory: ${category}`
-            })
+			if (data.status === 201) {
```
Comment (plon-Susk7) [inline src/components/Submission.jsx:36]: A failure message in the else statement would be perfect.

**Review by plon-Susk7 — APPROVED** [review]

File: src/components/Submission.jsx (hunk 1 above)
Comment (Yuvraj-Achrekar) [reply src/components/Submission.jsx:36]: ok i will add that too

**Comment by Yuvraj-Achrekar** [comment]
Added a toast for failure, message would be from api response and if not then a default message added

**Comment by plon-Susk7** [comment]
> Added a toast for failure, message would be from api response and if not then a default message added

Thanks for your contribution. Don't forget to star the repo :). Merging soon.

**Pull request merged**